corehq/apps/dump_reload/tests/test_sql_dump_load.py

```diff
@@ -150,6 +152,19 @@ def _check_signals_handle_raw(self, models):
                     receiver, model
                 )
                 self.assertIn('raw', args, message)
+                found_models.add(model)
+        expected_models = getattr(self, 'raw_post_save_models', set())
+        assert found_models == expected_models, "Found 'post_save' signal" \
+            f" with 'raw' arg for {found_models}, expected {expected_models}"
```

Comment:
I should know this, but where are we actually setting this `raw` arg to True? It seems like it defaults to False, and it sounds like it useful when saving an object that has been deserialized as opposed to initialized via a python object? Which I could see being useful in dump/load code, but am not following the trail well.